### PR TITLE
Rename decorator types to be more readable

### PR DESCRIPTION
**Pull request by e00E**

### CODE REVIEW

File: driver/src/logging.rs [hunk 1]
```diff
@@ -29,21 +29,23 @@ pub fn init(filter: impl AsRef<str>) -> (Logger, GlobalLoggerGuard) {
 
 /// Uses one decorator for `Error` and `Critical` log messages and the other for
 /// the rest.
-pub struct CustomFormatter<T0, T1> {
-    err_decorator: T0,
-    rest_decorator: T1,
+pub struct CustomFormatter<ErrDecorator, RestDecorator> {
```
Comment (nlordell) [inline driver/src/logging.rs]: `LogDecorator`? I feel like `Rest` is a loaded word.

Comment (e00E) [reply driver/src/logging.rs]: I was just reusing the variable name as CamelCase. We can go to ErrorLogsDecorator and OtherLogsDecorator.

Comment (nlordell) [reply driver/src/logging.rs]: Hm, I don’t have a good name for this. Just leave it as is I think. It’s descriptive enough.

Comment (nlordell) [reply driver/src/logging.rs]: Hm, I don’t have a good name for this. Just leave it as is I think. It’s descriptive enough.

**Pull request merged**